synapse/handlers/federation_event.py

```diff
@@ -92,6 +92,25 @@
     "Events received over federation that we marked as soft_failed",
 )
 
+backfill_processing_after_timer = Histogram(
+    "synapse_federation_backfill_processing_after_time_seconds",
+    "sec",
+    [],
+    buckets=(
+        1.0,
+        5.0,
+        10.0,
+        20.0,
+        30.0,
+        40.0,
+        60.0,
+        80.0,
+        120.0,
+        180.0,
+        "+Inf",
+    ),
```

Comment:
The after processing can take a while depending on how slow `/state_ids` is.

ex. `_process_pulled_events` taking 83s